test/e2e/mxGraph.model.test.ts

```diff
@@ -440,13 +439,15 @@ describe('mxGraph model', () => {
       label: 'End Event In Sub-Process',
       parentId: 'expanded_embedded_sub_process_id',
     });
-    expectModelContainsEdge('sequence_flow_in_sub_process_1_id', {
+    expect('sequence_flow_in_sub_process_1_id').toBeEdge({
       kind: FlowKind.SEQUENCE_FLOW,
       parentId: 'expanded_embedded_sub_process_id',
+      endArrow: 'blockThin',
```

Comment:
:question: is it possible not having to pass this? I guess that all sequence flows should end with this arrow, so why not hiding this?
This currently force to duplicate the value. Can we at least add a constant?
Or is this managed in #743?

Reply:
It's the case with toBeSequenceFlow, but here we don't use this method.